### PR TITLE
fix: Fixes TypeScript transpilation to ES5

### PR DESCRIPTION
**Pull request by caalador**
Fix the transpilation of TypeScript files
to ES5 for use with IE11

Part of #11713

### CODE REVIEW

File: flow-lit-template/src/main/java/com/vaadin/flow/component/littemplate/BundleLitParser.java [hunk 1]
```diff
@@ -182,6 +182,11 @@ private static String getSourceFromObject(JsonObject module,
                 && validKey(module, SOURCE, STRING)) {
             String name = module.getString(NAME);
 
+            // If the found module is
```
Comment (mshabarov) [inline flow-lit-template/src/main/java/com/vaadin/flow/component/littemplate/BundleLitParser.java:185]: is ES5 module?

File: flow-server/src/main/java/com/vaadin/flow/server/frontend/NodeUpdater.java [hunk 2]
```diff
@@ -308,10 +308,12 @@ static void addVaadinDefaultsToJson(JsonObject json) {
         defaults.put("webpack", "4.42.0");
         defaults.put("webpack-cli", "3.3.11");
         defaults.put("webpack-dev-server", "3.11.0");
-        defaults.put("webpack-babel-multi-target-plugin", "2.3.3");
+        defaults.put("webpack-babel-multi-target-plugin", "2.5.0");
```
Comment (mshabarov) [inline flow-server/src/main/java/com/vaadin/flow/server/frontend/NodeUpdater.java:311]: So the transpilation doesn't work in 2.3.3, but works in 2.5.0?

File: flow-lit-template/src/main/java/com/vaadin/flow/component/littemplate/BundleLitParser.java (hunk 1 above)
Comment (caalador) [reply flow-lit-template/src/main/java/com/vaadin/flow/component/littemplate/BundleLitParser.java]: It's what we have in BundleParser and it did leave me a bit confused. Elaborated on comment.

File: flow-server/src/main/java/com/vaadin/flow/server/frontend/NodeUpdater.java (hunk 2 above)
Comment (caalador) [reply flow-server/src/main/java/com/vaadin/flow/server/frontend/NodeUpdater.java:311]: The new configuration would seem to work with 2.3.3 also

**Review by mshabarov — APPROVED** [review]

File: flow-server/src/main/java/com/vaadin/flow/server/frontend/NodeUpdater.java (hunk 2 above)
Comment (mshabarov) [reply flow-server/src/main/java/com/vaadin/flow/server/frontend/NodeUpdater.java:311]: ok, so this version bump is not needed for this fix?
I have nothing against versions update, let's upgrade it anyway.

**Pull request merged**